pxtsim/localization.ts

```diff
@@ -0,0 +1,85 @@
+namespace pxsim.localization {
+    let _localizeStrings: Map<string> = {};
```

Comment:
Can we leave a comment here describing where to find the duplicated function. Just so we always update one and the other hand in hand